### PR TITLE
Change planar shadow func name v3.5.0

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: cocos-creator/3d-tasks#

Changelog:
 *

### CODE REVIEW

File: editor/i18n/en/localization.js [hunk 1]
```diff
@@ -100,8 +100,8 @@ module.exports = {
     },
     shadow: {
         enabled: 'Enable or disable real time shadows.',
-        normal: 'The normal of the plane which receives shadow.',
-        distance: 'The distance from coordinate origin to the receiving plane.',
+        planeDirection: 'The normal of the plane which receives shadow.',
```
Comment (pandamicro) [inline editor/i18n/en/localization.js]: The normal vector of the plane which receives shadow

File: editor/i18n/en/localization.js [hunk 2]
```diff
@@ -100,8 +100,8 @@ module.exports = {
...
+        planeHeight: 'Lets you adjust the height of a planar shadow.',
```
Comment (pandamicro) [inline editor/i18n/en/localization.js]: 保持一致的句式

The height from the origin of the plane which receives shadow

File: editor/i18n/en/localization.js [hunk 3]
```diff
@@ -113,6 +113,7 @@ module.exports = {
         orthoSize: 'Fix area size, the larger value, the lower precision of shadows.',
         invisibleOcclusionRange: 'If some shadow near the camera is missing, increase this value (world space unit) to fix it.',
         shadowDistance: 'Shadows do not appear beyond this distance (world space unit).',
+        maxReceived: 'Number of effective light sources that produce shadows',
```
Comment (pandamicro) [inline editor/i18n/en/localization.js]: Number of the effective light sources that produce shadows

File: editor/i18n/zh/localization.js [hunk 4]
```diff
@@ -100,8 +100,8 @@ module.exports = {
     },
     shadow: {
         enabled: '是否开启实时阴影',
-        normal: '阴影接收平面的法线，垂直于阴影，用于调整阴影的倾斜度',
-        distance: '阴影接收平面与原点的距离',
+        planeDirection: '阴影接收平面的法线，垂直于阴影，用于调整阴影的倾斜度',
+        planeHeight: '用于调整平面阴影的高度',
```
Comment (pandamicro) [inline editor/i18n/zh/localization.js]: 阴影接收平面距离原点的高度

**Review by pandamicro — APPROVED** [review]

**Review by jk20012001 — APPROVED** [review]

**Pull request merged**